crawl-ref/source/mon-gear.cc

```diff
@@ -925,6 +926,16 @@ int make_mons_weapon(monster_type type, int level, bool melee_only)
               { SPWPN_DISTORTION,      2 },
               { SPWPN_CHAOS,           1 }, },
         } },
+        { MONS_VAUD, {
+            { { WPN_FLAIL,          35},    //Total weight 100
+              { WPN_MORNINGSTAR,    24},
+              { WPN_LONG_SWORD,     20},
+              { WPN_SCIMITAR,       20},
+              { WPN_TRISHULA,       1},},
```

Comment:
Very spicy. I like it :) Probably people will complain about broken monster-player symmetry, though.